web/src/components/logs-table.css

```
@@ -1,47 +1,62 @@
 .lv-plugin__table .lv-plugin__table__expand {
-  font-size: var(--pf-v5-global--FontSize--sm);
-  --pf-v5-c-table--cell--Width: 50px;
-  --pf-v5-c-table--cell--MaxWidth: 50px;
-  --pf-v5-c-table--cell--MinWidth: 0px;
+  font-size: var(--pf-t--global--font--size--sm);
+  --pf-v6-c-table--cell--Width: 50px;
+  --pf-v6-c-table--cell--MaxWidth: 50px;
+  --pf-v6-c-table--cell--MinWidth: 0px;
+  font-size: var(--pf-t--global--font--size--sm);
+  --pf-v6-c-table--cell--Width: 50px;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "--pf-v6-c-table--cell--Width" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)